x/game/types/game.go

```diff
@@ -175,12 +177,11 @@ func (g *Game) Move(player string, populace uint32, direction Direction, amount
 	// abandoning a settlement with the exception of a capital which
 	// must always be occupied (in conflicts over a capital, there can
 	// never be a draw)
-	if pop.Amount == amount && pop.Settlement == Settlement_CAPITAL {
-		return sdkerrors.Wrap(ErrAbandoningCapital, pop.Position.String())
+	if pop.Amount <= amount {
+		amount = pop.Amount - 1
 	}
```

Comment:
Are you trying to make it that you always leave behind one population? I was thinking of making that change just in the UI. Like you press a hotkey and a move button and it moves all your population but 1. Actually Noam suggested that